src/index.html

```diff
@@ -5,8 +5,435 @@
+                data-qa="hover"
+                href="#"
+                class="card__button"
+              >
+                Buy
+              </a>
+            </div>
+          </div>
+        </div>
+        <div class="page__card card" data-qa="card">
+          <div class=" card__container">
+            <div class="card__photo card__photo--margin">
+              <img
+                class="card__image"
+                src="/src/images/imac.jpeg"
+                alt="imac"
+              >
+            </div>
+            <div class="card__title card__title--margin">
+              <p>APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+            </div>
+            <div class="card__subtitle card__subtitle--margin">
+              <p>Product code: 195434</p>
+            </div>
+            <div class="card__feedback feedback feedback--margin">
+              <div class="feedback__stars stars">
+                <div class="stars__star stars__star--1"></div>
+                <div class="stars__star stars__star--2"></div>
+                <div class="stars__star stars__star--3"></div>
+                <div class="stars__star stars__star--4"></div>
+                <div class="stars__star stars__star--5"></div>
+              </div>
+              <div class="feedback__review">
+                <p>Reviews: 5</p>
+              </div>
+            </div>
+            <div class="card__price price price--margin">
+              <div class="price__title">
+                <p>Price:</p>
+              </div>
+              <div class="price__amount">
+                <p>$2,199</p>
+              </div>
+            </div>
+            <div class="card__btn">
+              <a
+                data-qa="hover"
+                href="#"
+                class="card__button"
+              >
+                Buy
+              </a>
+            </div>
+          </div>
+        </div>
+        <div class="page__card card" data-qa="card">
+          <div class=" card__container">
+            <div class="card__photo card__photo--margin">
+              <img
+                class="card__image"
+                src="/src/images/imac.jpeg"
+                alt="imac"
+              >
+            </div>
+            <div class="card__title card__title--margin">
+              <p>APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+            </div>
+            <div class="card__subtitle card__subtitle--margin">
+              <p>Product code: 195434</p>
+            </div>
+            <div class="card__feedback feedback feedback--margin">
+              <div class="feedback__stars stars">
+                <div class="stars__star stars__star--1"></div>
+                <div class="stars__star stars__star--2"></div>
+                <div class="stars__star stars__star--3"></div>
+                <div class="stars__star stars__star--4"></div>
+                <div class="stars__star stars__star--5"></div>
+              </div>
```

Comment:
```suggestion
              <div class="feedback__stars stars stars--4">
                <div class="stars__star"></div>
                <div class="stars__star"></div>
                <div class="stars__star"></div>
                <div class="stars__star"></div>
                <div class="stars__star"></div>
              </div>
```

Reply:
you shouldn't count every start, only u need to set number of active stars in modifier for 
`stars` block